### PR TITLE
update links and fix tests

### PR DESCRIPTION
**Pull request by vutran**
- Fix link references to new repo
- Mock `download` module as it was failing the tests

### CODE REVIEW

**Comment by vutran** [comment]
coveralls not allowing me to add dextapp repo (maybe a caching issue on their server).

**Review by adnasa — APPROVED** [review]
Thanks for doing this!
although, this would cause heck of a merge conflict with #184 

any chance you can cherry-pick this into that branch? then we just merge that branch 😉

**Comment by vutran** [comment]
Yeah I'll wait for yours

**Comment by adnasa** [comment]
If only I can merge tho 😂

**Comment by adnasa** [comment]
hmmm... weirdly enough, snapshot tests fails? I'll have a look

File: __tests__/app/renderer/src/reducers/copiedToClipboard.js [hunk 1]
```diff
@@ -34,8 +34,8 @@ describe('copiedToClipboard reducer', () => {
 
   it('should handle the rest of the actions', () => {
     const state = undefined;
-    Object.values(allTypes).map(type =>
```
Comment (adnasa) [inline __tests__/app/renderer/src/reducers/copiedToClipboard.js:37]: if I remember correctly, this requires `stage-3` to work again. But I wouldn't worry too much about this right. We'll deal with it in conjunction with upgrading `babel`

File: __tests__/app/renderer/src/components/__snapshots__/QueryField.js.snap [hunk 2]
```diff
@@ -1,19 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
 exports[`<QueryField /> component should render a query field component with no value 1`] = `
 <div
-  data-css-1ykjhyy="">
```
Comment (adnasa) [inline __tests__/app/renderer/src/components/__snapshots__/QueryField.js.snap:3]: I didn't think formatting would affect snapshots like this. my bad

**Review by adnasa — APPROVED** [review]
perfect, code wise

re-approved 😉 
I did a grep-ish on this locally, and everything is fine. 

merge it"

File: __tests__/app/renderer/src/reducers/copiedToClipboard.js (hunk 1 above)
Comment (vutran) [reply __tests__/app/renderer/src/reducers/copiedToClipboard.js:37]: 👍

**Pull request merged**